lti_consumer/lti_1p3/consumer.py

```diff
@@ -161,6 +163,47 @@ def set_launch_presentation_claim(
             },
         }
 
+    def set_context_claim(
+            self,
+            context_id,
+            context_types=None,
+            context_title=None,
+            context_label=None
+    ):
+        """
+        Optional: Set context claims
+
+        https://www.imsglobal.org/spec/lti/v1p3/#context-claim
+
+        Arguments:
+            context_id (string):  Unique value identifying the user
+            context_types (list):  A list of context type values for the claim
+            context_title (string):  Plain text title of the context
+            context_label (string):  Plain text label for the context
+        """
+        if not isinstance(context_types, list):
+            context_types = [context_types]
+
+        context_claim_types = [
```

Comment:
Can you add a comment here explicitly stating that all custom contexts are not being included in the context claim.

Reply:
Done in the next commit